misc/requirements/requirements-tests.txt-raw

```diff
@@ -39,7 +39,6 @@ tldextract
 # Include them here even though we don't need them to make sure we at least
 # get an up to date version.
 importlib_resources
-#@ markers: importlib_resources python_version=="3.8.*"
```

Comment:
Given that `importlib_resources` makes its way into this file *somehow* (not quite sure yet it's via pip, as it's not in the other files?!), this marker is probably incorrect, so I removed it.